src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/Validation/ValidationRulesAccess.swift

```diff
@@ -0,0 +1,19 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import SwiftCBOR
+import CertLogic
+
+struct ValidationRulesAccess {
```

Comment:
missing `public`, otherwise I see an swift lint error

Reply:
fixed and added public init.